frontend/src/index.ts

```diff
@@ -243,58 +294,40 @@ export class App extends LiteElement {
               `
             : ""}
 
-          <div class="grid grid-flow-col gap-3 md:gap-5 items-center">
+          <div class="grid grid-flow-col auto-cols-max gap-3 items-center">
             ${this.authService.authState
-              ? html` <sl-dropdown placement="bottom-end">
-                  <sl-icon-button
-                    slot="trigger"
-                    name="person-circle"
-                    style="font-size: 1.5rem;"
-                  ></sl-icon-button>
-
-                  <sl-menu class="w-60 min-w-min max-w-full">
-                    <div class="px-7 py-2">
-                      ${isAdmin
-                        ? html`
-                            <div class="mb-2">
-                              <sl-tag
-                                class="uppercase"
-                                variant="primary"
-                                size="small"
-                                >${msg("admin")}</sl-tag
-                              >
-                            </div>
-                          `
+              ? html` ${this.renderTeams()}
+                  <sl-dropdown placement="bottom-end">
+                    <sl-icon-button
+                      slot="trigger"
+                      name="person-circle"
+                      style="font-size: 1.5rem;"
+                    ></sl-icon-button>
+
+                    <sl-menu class="w-60 min-w-min max-w-full">
+                      <div class="px-7 py-2">${this.renderMenuUserInfo()}</div>
+                      <sl-divider></sl-divider>
+                      <sl-menu-item
+                        @click=${() => this.navigate(ROUTES.accountSettings)}
+                      >
+                        <sl-icon slot="prefix" name="gear"></sl-icon>
+                        ${msg("Your Account")}
```

Comment:
Can we unify this label with the name of the page it goes to?
```suggestion
                        ${msg("Account Settings")}
```